Antidote/IncomingCallNotificationView.m

```diff
@@ -0,0 +1,148 @@
+//
+//  IncomingCallNotificationView.m
+//  Antidote
+//
+//  Created by Chuong Vu on 7/21/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import "IncomingCallNotificationView.h"
+#import "AppearanceManager.h"
+#import "Masonry.h"
+
+static const CGFloat kIncomingNameFontSize = 12.0;
+static const CGFloat kIncomingIsCallingFontSize = 10.0;
+static const CGFloat kButtonSize = 50.0;
+static const CGFloat kButtonBorderWidth = 1.0;
+static const CGFloat kIndent = 20.0;
+
+@interface IncomingCallNotificationView ()
+
+@property (strong, nonatomic) NSString *nickname;
+
+@property (strong, nonatomic) UILabel *nameLabel;
+@property (strong, nonatomic) UILabel *descriptionLabel;
+@property (strong, nonatomic) UIButton *declineButton;
+@property (strong, nonatomic) UIButton *acceptButton;
+
+@end
+@implementation IncomingCallNotificationView
+
+
+#pragma mark - LifeCycle
+- (instancetype)initWithNickname:(NSString *)nickname
+{
+    self = [super init];
+    if (! self) {
+        return nil;
+    }
+
+    _nickname = nickname;
+
+    self.backgroundColor = [UIColor grayColor];
+
+    [self createNameLabel];
```

Comment:
There is no need in `nickname` property, you can simply pass nickname string to `createNameLabel` method.